client/components/pie-chart/index.js

```diff
@@ -8,9 +8,9 @@ import DataType from './data-type';
 
 import './style.scss';
 
-const SVG_SIZE = 300;
+const SVG_SIZE = 224;
 const NUM_COLOR_SECTIONS = 3;
-const TOOLTIP_OFFSET_X = -207;
+const TOOLTIP_OFFSET_X = -164; // SVG_SIZE - 2x tip offset (30px in CSS)
```

Comment:
@dognose24  do you remember how `207` was calculated? I can't get it to work for the new width. Maybe the issue is overlapping with the width mentioned above.

Reply:
Since the tooltip's left position is adjusted by `coordinates[ 0 ] + svgSize / 2` with the SVG size, I think this constant should not be modified because it is more relevant to the tooltip width.

I also found that the tooltip position is not accurate at the moment:

https://github.com/Automattic/wp-calypso/assets/6869813/7673eba0-adf5-4e54-ba8c-5370492ef603